.github/workflows/build-multiarch.yaml

```diff
@@ -96,15 +96,14 @@ jobs:
           images: |
             ${{ env.REGISTRY_BASE }}/govuk-ruby-base
           tags: |
-            type=semver,pattern={{raw}},value=${{ join(matrix.version.rubyver, '.') }}
             type=raw,value=${{ matrix.version.rubyver[0] }}.${{ matrix.version.rubyver[1] }}
             type=raw,value=latest,enable=${{ matrix.version.extra == 'latest' }}
-            type=sha,enable=true,prefix=${{ join(matrix.version.rubyver, '.') }}-,format=short
-            type=sha,enable=true,priority=100,format=long,prefix=${{ join(matrix.version.rubyver, '.') }}-
+            type=sha,enable=true,prefix=${{ matrix.version.rubyver[0] }}.${{ matrix.version.rubyver[1] }}-,format=short
+            type=sha,enable=true,priority=100,format=long,prefix=${{ matrix.version.rubyver[0] }}.${{ matrix.version.rubyver[1] }}-
```

Comment:
note for reviewers: this (-3, +2) diff hunk should be identical in each of the four cases (unfortunately the labels are repeated 4 times and I couldn't quickly see a way to clean that up — suggestions welcome though!)